flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/log/CygnusLogger.java

```diff
@@ -76,7 +112,14 @@ public void warn(String componentName, String message) {
      * @param message
      */
     public void debug(String componentName, String message) {
-        debug("[" + componentName + "] " + message);
+        try {
+            debug("[" + componentName + "] " + message);
+        } catch (Exception e) {
+            if (shutdown) {
+                System.err.println("A problem with the logging system was found... shutting down Cygnus right now!");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/083ba577850ad21c798abe1c183f794ee2f2c4ea